hedera-node/hedera-app/src/main/java/com/hedera/node/app/workflows/handle/validation/StandardizedAttributeValidator.java

```diff
@@ -0,0 +1,139 @@
+/*
+ * Copyright (C) 2023 Hedera Hashgraph, LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.hedera.node.app.workflows.handle.validation;
+
+import static com.hedera.hapi.node.base.ResponseCodeEnum.AUTORENEW_DURATION_NOT_IN_RANGE;
+import static com.hedera.hapi.node.base.ResponseCodeEnum.BAD_ENCODING;
+import static com.hedera.hapi.node.base.ResponseCodeEnum.INVALID_EXPIRATION_TIME;
+import static com.hedera.hapi.node.base.ResponseCodeEnum.INVALID_ZERO_BYTE_IN_STRING;
+import static com.hedera.hapi.node.base.ResponseCodeEnum.MEMO_TOO_LONG;
+import static com.hedera.node.app.spi.config.PropertyNames.ENTITIES_MAX_LIFETIME;
+import static com.hedera.node.app.spi.workflows.HandleException.validateTrue;
+import static java.util.Objects.requireNonNull;
+
+import com.hedera.hapi.node.base.Key;
+import com.hedera.node.app.service.mono.context.annotations.CompositeProps;
+import com.hedera.node.app.service.mono.context.properties.GlobalDynamicProperties;
+import com.hedera.node.app.service.mono.context.properties.PropertySource;
+import com.hedera.node.app.spi.validation.AttributeValidator;
+import com.hedera.node.app.spi.workflows.HandleException;
+import edu.umd.cs.findbugs.annotations.NonNull;
+import java.nio.charset.StandardCharsets;
+import java.util.function.LongSupplier;
+import javax.inject.Inject;
+import javax.inject.Singleton;
+
+/**
+ * An implementation of {@link AttributeValidator} that encapsulates the current policies for
+ * validating attributes of entities, <i>without</i> any use of {@code mono-service} code.
+ */
+@Singleton
+public class StandardizedAttributeValidator implements AttributeValidator {
+    public static final int MAX_NESTED_KEY_LEVELS = 15;
+
+    private final long maxEntityLifetime;
+    private final LongSupplier consensusSecondNow;
+    private final GlobalDynamicProperties dynamicProperties;
+
+    @Inject
+    public StandardizedAttributeValidator(
+            @NonNull final LongSupplier consensusSecondNow,
+            @NonNull final @CompositeProps PropertySource properties,
```

Comment:
Having 1 annotation before final and the other one after `final` makes it harder to read from my point of view

Reply:
Put both first.